### ISSUE TITLE
ECO mode cause board to constantly reboot when sleep time set above 2100 seconds

### ISSUE THREAD

**Issue by Marvin-1802** [kike-canaries/canairio_android#156]
**the bug**

I was testing a board outdoors running on a battery and solar panel, using ECO mode set at 1800 seconds (30 min) After several days I switched the board to go into ECO mode for 3600 seconds (60 min).  The battery started to drain very quickly, and I discovered that the board was stuck in a reboot loop.  When I changed the sleep time to 2100 seconds the issue went away.  I tested at 2200, 2500, 3000, 3500 seconds and the reboot loop started again.

_This is on a TTGO T-Display using CanAirIO Bike with AHT10 Temp + Humi sensor_

**Comment by roberbike** [comment]
Can you post here the log about reboot loop?. I did not test this ECO time with such high values.

**Comment by Marvin-1802** [comment]
Yes, it is a long sample rate for AQI data.  With the cold weather here, and the short daylight hours I was trying to extend my battery/solar.  
Loop Log attached>
[esp-web-tools-logs.txt](https://github.com/kike-canaries/canairio_android/files/10230284/esp-web-tools-logs.txt)

**Comment by roberbike** [comment]
To solve this issue, I made some changues in my own compilation:
In platformio.ini 
[env:TTGO_TDISPLAY]
extends = esp32_common
board = esp32dev
lib_deps = 
	${common.lib_deps}
	bodmer/TFT_eSPI @ 2.4.70
lib_ignore = 
	gui-utils-oled
build_flags = 
	${common.build_flags}
	-Os
	-DUSER_SETUP_LOADED=1
	-DST7789_DRIVER=1
	-DENABLE_TFT=1
	-DTFT_WIDTH=135
	-DTFT_HEIGHT=240
	-DCGRAM_OFFSET=1
	-DTFT_MOSI=19
	-DTFT_SCLK=18
	-DTFT_CS=5
	-DTFT_DC=16
	-DTFT_RST=23
	-DTFT_BL=4
	-DTOUCH_CS=-1
	-DTFT_BACKLIGHT_ON=1
	-DLOAD_GLCD=1
	-DLOAD_FONT2=1
	-DLOAD_FONT4=1
	-DLOAD_FONT6=1
	-DLOAD_FONT7=1
	-DLOAD_FONT8=1
	-DLOAD_GFXFF=1
	-DSMOOTH_FONT=1
	-DSPI_FREQUENCY=40000000
	-DSPI_READ_FREQUENCY=6000000

In /scr/power.cpp

line 50          esp_sleep_enable_timer_wakeup(seconds * 1000000ull);

And it is working with 2 examples. ! with 2125 seg and other with 3150 seg of deep-sleep.

![image](https://user-images.githubusercontent.com/20878412/208237018-dd4a5749-19ff-4bd8-80f8-57158b0ef1be.png)

**Comment by roberbike** [comment]
Solved with ver 922, devel.
